Extension/src/main.ts

```diff
@@ -176,25 +179,25 @@ export function deactivate(): Thenable<void> {
 }
 
 function removePotentialPII(str: string): string {
-    let words = str.split(" ");
-    let result = "";
+    let words: string[] = str.split(" ");
+    let result: string = "";
     for (let word of words) {
         if (word.indexOf(".") == -1 && word.indexOf("/") == -1 && word.indexOf("\\") == -1 && word.indexOf(":") == -1) {
             result += word + " ";
         }
         else {
```

Comment:
is there a rule to enforce correct formatting of if-else blocks?  The "else" should be on the line above next to the `{`